lambdas/layer/Makefile

```diff
@@ -8,10 +8,6 @@ include ../lambdas.mk
 wheels: check_env
 	rm -rf vendor/jsonobject*
```

Comment:
This needs to be changed to remove the whole contents of vendor or else add upgrading instructions to do that once so that we don't keep packaging the old wheels redundantly.

Reply:
We should remove the whole contents of vendor because this caused a problem for @danielsotirhos where pyresist somehow ended up in there and caused duplicates.